File: Swift4/Swift4/model/Igrac.swift
//
//  Igrac.swift
//  Swift4
//
//  Created by Tomislav Jakopec on 22/02/2020.
//  Copyright © 2020 Tomislav Jakopec. All rights reserved.
//

import Foundation
class Igrac: Entitet{
    private var ime: String
    private var prezime: String
    private var urlSlika: String
    private var spol: Int
    
    init(sifra: Int = 0, ime: String, prezime: String, urlSlika: String, spol: Int) {
        self.ime = ime
        self.prezime = prezime
        self.urlSlika = urlSlika
        self.spol = spol
        super.init(sifra: sifra)
    }
    
    public func getIme()->String{
        return self.ime
    }
    
    public func setIme(ime: String){
        self.ime = ime
    }
    
    public func getPrezime()->String{
        return self.prezime
    }
    
    public func setPrezime(prezime: String){
        self.prezime = prezime
    }
    
    public func getUrlSlika()->String{
        return self.urlSlika
    }
    
    public func setUrlSlika(urlSlika: String){
        self.urlSlika = urlSlika
    }
    
    public func getSpol()->Int{
        return self.spol
    }
    
    public func setSpol(spol: Int){
        self.spol = spol
    }
    
    
    public var description: String {
        return self.ime + " " + self.prezime
    }
    
    
}

File: Java/src/jakopec/model/Igrac.java
package jakopec.model;

public class Igrac extends Entitet {

    private String ime;
    private String prezime;
    private String urlSlika;
    private int spol;

    public Igrac() {
    }

    public Igrac(int sifra, String ime, String prezime, String urlSlika, int spol) {
        super(sifra);
        this.ime = ime;
        this.prezime = prezime;
        this.urlSlika = urlSlika;
        this.spol = spol;
    }

    public String getIme() {
        return ime;
    }

    public void setIme(String ime) {
        this.ime = ime;
    }

    public String getPrezime() {
        return prezime;
    }

    public void setPrezime(String prezime) {
        this.prezime = prezime;
    }

    public String getUrlSlika() {
        return urlSlika;
    }

    public void setUrlSlika(String urlSlika) {
        this.urlSlika = urlSlika;
    }

    public int getSpol() {
        return spol;
    }

    public void setSpol(int spol) {
        this.spol = spol;
    }

    @Override
    public String toString() {
        return ime + " " + prezime;
    }
}
